tox.ini

```diff
@@ -4,7 +4,7 @@
 [tox]
 no_package = True
 skip_missing_interpreters = True
-env_list = lint, unit
+env_list = lint, unit, integration-{lxd,k8s}-{kafka,mysql,mongodb,postgresql}
```

Comment:
By default I would say it would be more fair to run everything if one does not provide any argument. But i'm not too firm on this. I don't like discrimination of integrationt tests :joy: 

Anyhow, i'll revert this, as I don't like things to be non-standard if everything else is otherwise